server/rooms.ts

```diff
@@ -986,7 +986,12 @@ export class GlobalRoom extends BasicRoom {
 			}
 		}
 		const stack = stackLines.slice(0, 2).join(`<br />`);
-		const crashMessage = `|html|<div class="broadcast-red"><b>${crasher} has crashed:</b> ${stack}</div>`;
+		let crashMessage;
+		if (/private|chat-plugins&#x2f;_/.test(stack)) {
```

Comment:
```suggestion
		if (/private/.test(stack)) {
```
If `_` hasn't been fixed yet, it will be. Either way, `_` as a privacy marker is not something I want to support.

Reply:
I'd also probably move the notification to the staff or upper staff room instead of hiding it completely.